projects/canopy/src/lib/table/table-cell/table-cell.component.scss

```diff
@@ -5,6 +5,26 @@
   display: flex;
   align-items: center;
 
+  &--stacked {
```

Comment:
Because the input is now `stack` should the class also be `--stack`?

Reply:
Really sorry Elena, I didn't see this message before I hit merge. Given that we no longer officially support flat HTML, perhaps changing underlaying classes doesn't constitute as a breaking change anymore? If so - I could change this in a new PR. What do you think?

Reply:
I think that's totally fine. I don't think it's a breaking change :)